[2024-03-03T03:35:24.145+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_1 scheduled__2016-04-03T00:00:00+00:00 [queued]>
[2024-03-03T03:35:24.159+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_1 scheduled__2016-04-03T00:00:00+00:00 [queued]>
[2024-03-03T03:35:24.160+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:35:24.182+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_1> on 2016-04-03 00:00:00+00:00
[2024-03-03T03:35:24.190+0000] {standard_task_runner.py:60} INFO - Started process 1463 to run task
[2024-03-03T03:35:24.195+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_1', 'scheduled__2016-04-03T00:00:00+00:00', '--job-id', '560', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmp3ortucdl']
[2024-03-03T03:35:24.198+0000] {standard_task_runner.py:88} INFO - Job 560: Subtask get_files_1
[2024-03-03T03:35:24.228+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:35:24.299+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_1 scheduled__2016-04-03T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:35:24.457+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_1' AIRFLOW_CTX_EXECUTION_DATE='2016-04-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2016-04-03T00:00:00+00:00'
[2024-03-03T03:35:24.460+0000] {logging_mixin.py:188} INFO - [1901, 1902, 1903, 1904, 1905, 1906, 1907, 1908, 1909, 1910, 1911, 1912, 1913, 1914, 1915, 1916, 1917, 1918, 1919, 1920, 1921]
[2024-03-03T03:35:24.462+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:35:24.479+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_1, execution_date=20160403T000000, start_date=20240303T033524, end_date=20240303T033524
[2024-03-03T03:35:24.529+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:35:24.556+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
